mapper.html

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
Could you add a screenshot of this page to the PR Conversation thread?

(It's not necessary to add it as part of the commit, but you can if you want to)

Reply:
added to the description